package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "nodejs_epam",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js",
```

Comment:
Use ‘./node_modules/.bin/nodemon index.js’, please. It would be better to avoid installing nodemon globally.